### ISSUE TITLE
Error Regarding Missing Files for WebmConverter

### ISSUE THREAD

**Issue by PlasmaLover** [argorar/WebMConverter#76]
Hello, I created a GitHub account for this since I was unable to find a solution from online. I have pasted below the errors I have gotten. I have tried all the following including:
- downloading .NET Framework 4.8
- latest AviSynth (2.6.0, 32-bit) and also the newer version to to try it out 
 
I have also reinstalled the program as well but I am unsure what is causing this issue. I have never had an issue with WebmConverter for the past 4 yrs I have been using it until now. I would appreciate any help.

I have left the [Processing] tab unchanged. I have included a screenshot of the only changes I've inputed for the program.
Below is the error code I had as well as the options I was trying to fix.

=======

Arguments for pass 1: -f nut -i pipe:0   -an -c:v libvpx-vp9 -pix_fmt yuv420p -threads 12 -slices 8 -metadata title="testvid" -lag-in-frames 16 -auto-alt-ref 1  -b:v 1389k -qcomp 0 -fs 4194304 -aq-mode 4 -row-mt 1 -tile-columns 6 -tile-rows 2 -pass 1 -passlogfile "C:\Users\Plasma User\AppData\Local\Temp\aa9cb5d7-338e-45d7-869f-377614b47d8f"  -f webm -y "NUL"
Arguments for pass 2: -f nut -i pipe:0   -an -c:v libvpx-vp9 -pix_fmt yuv420p -threads 12 -slices 8 -metadata title="testvid" -lag-in-frames 16 -auto-alt-ref 1  -b:v 1389k -qcomp 0 -fs 4194304 -aq-mode 4 -row-mt 1 -tile-columns 6 -tile-rows 2 -pass 2 -passlogfile "C:\Users\Plasma User\AppData\Local\Temp\aa9cb5d7-338e-45d7-869f-377614b47d8f"  -f webm -y "E:\Jonathan\Videos\REcording\testvid.webm"

--- CREATING AVISYNTH PROXY --- 
C:\Users\Plasma User\AppData\Local\Temp\tmp1A4F.tmp: Unknown error occurred

[nut @ 0000012d6db7d3c0] No main startcode found.
[in#0 @ 0000012d6db7d280] Error opening input: Invalid data found when processing input
--- FFMPEG HAS EXITED ---
--- ENTERING PASS 2 ---
--- CREATING AVISYNTH PROXY --- 
Error opening input file pipe:0.
Error opening input files: Invalid data found when processing input
C:\Users\Plasma User\AppData\Local\Temp\tmp1A4F.tmp: Unknown error occurred

[nut @ 000001ea7f6bd500] No main startcode found.
[in#0 @ 000001ea7f6bd380] Error opening input: Invalid data found when processing input
Error opening input file pipe:0.
--- FFMPEG HAS EXITED ---
Error opening input files: Invalid data found when processing input

ffmpeg.exe exited with exit code -1094995529. That's usually bad.
If you have no idea what went wrong, open an issue on Github and copy paste the output of this window there, also upload test video.

![Screenshot 2024-05-07 010439](https://github.com/argorar/WebMConverter/assets/169137611/2bb8bc25-e79a-44fd-84ff-8a684a1a1093)

**Comment by argorar** [comment]
Try moving testvid.mp4 to disk C and try to convert.

**Comment by PlasmaLover** [comment]
I moved it to disk C and got an error of the similar sort still.

===

Arguments for pass 1: -f nut -i pipe:0   -an -c:v libvpx-vp9 -pix_fmt yuv420p -threads 12 -slices 8 -metadata title="testvid" -lag-in-frames 16 -auto-alt-ref 1  -b:v 1389k -qcomp 0 -fs 4194304 -aq-mode 4 -row-mt 1 -tile-columns 6 -tile-rows 2 -pass 1 -passlogfile "C:\Users\Plasma User\AppData\Local\Temp\ee659fe4-01f7-46c1-9a5d-c6363341c0ec"  -f webm -y "NUL"
Arguments for pass 2: -f nut -i pipe:0   -an -c:v libvpx-vp9 -pix_fmt yuv420p -threads 12 -slices 8 -metadata title="testvid" -lag-in-frames 16 -auto-alt-ref 1  -b:v 1389k -qcomp 0 -fs 4194304 -aq-mode 4 -row-mt 1 -tile-columns 6 -tile-rows 2 -pass 2 -passlogfile "C:\Users\Plasma User\AppData\Local\Temp\ee659fe4-01f7-46c1-9a5d-c6363341c0ec"  -f webm -y "C:\Users\Plasma User\Videos\Captures\testvid.webm"

--- CREATING AVISYNTH PROXY --- 
C:\Users\Plasma User\AppData\Local\Temp\tmpB53D.tmp: Unknown error occurred

[nut @ 000001e9d618d3c0] No main startcode found.
[in#0 @ 000001e9d618d280] Error opening input: Invalid data found when processing input
--- FFMPEG HAS EXITED ---
--- ENTERING PASS 2 ---
--- CREATING AVISYNTH PROXY --- 
Error opening input file pipe:0.
Error opening input files: Invalid data found when processing input
C:\Users\Plasma User\AppData\Local\Temp\tmpB53D.tmp: Unknown error occurred

[nut @ 000002b7966bd500] No main startcode found.
[in#0 @ 000002b7966bd380] Error opening input: Invalid data found when processing input
--- FFMPEG HAS EXITED ---
Error opening input file pipe:0.
Error opening input files: Invalid data found when processing input

ffmpeg.exe exited with exit code -1094995529. That's usually bad.
If you have no idea what went wrong, open an issue on Github and copy paste the output of this window there, also upload test video.

====

![Screenshot 2024-05-07 205926](https://github.com/argorar/WebMConverter/assets/169137611/72c37dd0-a61d-4c5e-9da6-8367a62d438e)

**Comment by PlasmaLover** [comment]
My current webmConverter is running in my disk C and ran it as an administrator.

I am currently running in avisynth 3.7.3

I am unable to load the tmp file but I will attach the the file it underneath this error log

![Screenshot 2024-05-08 130100](https://github.com/argorar/WebMConverter/assets/169137611/401850c0-ccce-4537-a4c4-dba89cce936e)
![Screenshot 2024-05-08 130121](https://github.com/argorar/WebMConverter/assets/169137611/e0dfb95d-d580-45d9-a829-7d7d59bc62f6)


====

Arguments for pass 1: -f nut -i pipe:0   -an -c:v libvpx-vp9 -pix_fmt yuv420p -threads 12 -slices 8 -metadata title="testvid" -lag-in-frames 16 -auto-alt-ref 1  -b:v 2778k -qcomp 0 -fs 8388608 -aq-mode 4 -row-mt 1 -tile-columns 6 -tile-rows 2 -pass 1 -passlogfile "C:\Users\Jonathan\AppData\Local\Temp\74b0fcb0-d210-49b7-8112-50fb46e28f49"  -f webm -y "NUL"
Arguments for pass 2: -f nut -i pipe:0   -an -c:v libvpx-vp9 -pix_fmt yuv420p -threads 12 -slices 8 -metadata title="testvid" -lag-in-frames 16 -auto-alt-ref 1  -b:v 2778k -qcomp 0 -fs 8388608 -aq-mode 4 -row-mt 1 -tile-columns 6 -tile-rows 2 -pass 2 -passlogfile "C:\Users\Jonathan\AppData\Local\Temp\74b0fcb0-d210-49b7-8112-50fb46e28f49"  -f webm -y "C:\Users\Plasma User\Documents\WebmforLazys\WebMConverter-3.39.0\testvid.webm"

--- CREATING AVISYNTH PROXY --- 
C:\Users\Jonathan\AppData\Local\Temp\tmp5A8D.tmp: Unknown error occurred

[nut @ 000001f0665dd2c0] No main startcode found.
[in#0 @ 000001f0665dd180] Error opening input: Invalid data found when processing input
Error opening input file pipe:0.
--- FFMPEG HAS EXITED ---
--- ENTERING PASS 2 ---
--- CREATING AVISYNTH PROXY --- 
Error opening input files: Invalid data found when processing input
C:\Users\Jonathan\AppData\Local\Temp\tmp5A8D.tmp: Unknown error occurred

[nut @ 000001d5ad5ad340] No main startcode found.
[in#0 @ 000001d5ad5ad1c0] Error opening input: Invalid data found when processing input
Error opening input file pipe:0.
--- FFMPEG HAS EXITED ---
Error opening input files: Invalid data found when processing input

ffmpeg.exe exited with exit code -1094995529. That's usually bad.
If you have no idea what went wrong, open an issue on Github and copy paste the output of this window there, also upload test video.


==================================================================================
And here is the tmp file and what it says:
==================================================================================


PluginPath = "\\?\C:\Users\PLASMA~1\DOCUME~1\WEBMFO~1\WEBMCO~1.0\Binaries\Win32\"
try { LoadPlugin(PluginPath+"ffms2.dll") } catch (_) { LoadCPlugin(PluginPath+"ffms2.dll") }
AudioDub(FFVideoSource("C:\Users\Plasma User\Documents\WebmforLazys\WebMConverter-3.39.0\testvid.mp4",cachefile="C:\Users\Jonathan\AppData\Local\Temp\BF-E3-B9-4D-2D-17-7F-55-AB-8B-18-C4-3B-8C-54-F5.ffindex",track=0), FFAudioSource("C:\Users\Plasma User\Documents\WebmforLazys\WebMConverter-3.39.0\testvid.mp4",cachefile="C:\Users\Jonathan\AppData\Local\Temp\BF-E3-B9-4D-2D-17-7F-55-AB-8B-18-C4-3B-8C-54-F5.ffindex",track=1))
# This is an AviSynth script. You may write advanced commands below, or just press the buttons above for smooth sailing.

**Comment by argorar** [comment]
Remove all the avisynth versions. Install [this one](https://sourceforge.net/projects/avisynth2/files/AviSynth%202.6/AviSynth%202.6.0/AviSynth_260.exe/download). 
To be sure, tested with other videos? What windows version do you have?

**Comment by PlasmaLover** [comment]
I have removed the newer Avisynth versions and installed the one you linked. 
I have also tested with another video and it still has given me the same error. 
My current windows is = Microsoft Windows 11 Enterprise and I have updated it a week ago.
I ran it as an admin as well as using the Avisynth link you gave me.

I have also included the tmp folder error here too.

![Screenshot 2024-05-08 225250](https://github.com/argorar/WebMConverter/assets/169137611/2edae32a-69d9-44e4-afcb-34dbc6dea388)

===============

![Screenshot 2024-05-08 225534](https://github.com/argorar/WebMConverter/assets/169137611/cc229d33-7b41-45a5-8fce-53a01e92d007)

===============

Arguments for pass 1: -f nut -i pipe:0   -an -c:v libvpx-vp9 -pix_fmt yuv420p -threads 12 -slices 4 -metadata title="testvid2" -lag-in-frames 16 -auto-alt-ref 1  -b:v 24430k -qcomp 0 -fs 52428800 -aq-mode 4 -row-mt 1 -tile-columns 6 -tile-rows 2 -pass 1 -passlogfile "C:\Users\Jonathan\AppData\Local\Temp\909b29c1-4b5d-4b24-bcbb-d233d0165d8d"  -f webm -y "NUL"
Arguments for pass 2: -f nut -i pipe:0   -an -c:v libvpx-vp9 -pix_fmt yuv420p -threads 12 -slices 4 -metadata title="testvid2" -lag-in-frames 16 -auto-alt-ref 1  -b:v 24430k -qcomp 0 -fs 52428800 -aq-mode 4 -row-mt 1 -tile-columns 6 -tile-rows 2 -pass 2 -passlogfile "C:\Users\Jonathan\AppData\Local\Temp\909b29c1-4b5d-4b24-bcbb-d233d0165d8d"  -f webm -y "C:\Users\Plasma User\Documents\WebmforLazys\WebMConverter-3.39.0\testvid2.webm"

--- CREATING AVISYNTH PROXY --- 
[avisynth @ 0286e4a0] Unable to load C Plugin: "\\?\C:\Users\PLASMA~1\DOCUME~1\WEBMFO~1\WEBMCO~1.0\Binaries\Win32\ffms2.dll", error=0x7e
(C:\Users\Jonathan\AppData\Local\Temp\tmp3680.tmp, line 2)
(C:\Users\Jonathan\AppData\Local\Temp\tmp3680.tmp, line 2)
[nut @ 000001673c63d280] No main startcode found.
C:\Users\Jonathan\AppData\Local\Temp\tmp3680.tmp: Unknown error occurred
[in#0 @ 000001673c63d140] Error opening input: Invalid data found when processing input
--- FFMPEG HAS EXITED ---
--- ENTERING PASS 2 ---
--- CREATING AVISYNTH PROXY --- 

Error opening input file pipe:0.
Error opening input files: Invalid data found when processing input
[avisynth @ 0279e4a0] Unable to load C Plugin: "\\?\C:\Users\PLASMA~1\DOCUME~1\WEBMFO~1\WEBMCO~1.0\Binaries\Win32\ffms2.dll", error=0x7e
[nut @ 0000015814f9d1c0] No main startcode found.
(C:\Users\Jonathan\AppData\Local\Temp\tmp3680.tmp, line 2)
--- FFMPEG HAS EXITED ---
[in#0 @ 0000015814f9d080] Error opening input: Invalid data found when processing input
(C:\Users\Jonathan\AppData\Local\Temp\tmp3680.tmp, line 2)
Error opening input file pipe:0.
C:\Users\Jonathan\AppData\Local\Temp\tmp3680.tmp: Unknown error occurred
Error opening input files: Invalid data found when processing input


ffmpeg.exe exited with exit code -1094995529. That's usually bad.
If you have no idea what went wrong, open an issue on Github and copy paste the output of this window there, also upload test video.

=======
Temp folder error:
=========

PluginPath = "\\?\C:\Users\PLASMA~1\DOCUME~1\WEBMFO~1\WEBMCO~1.0\Binaries\Win32\"
try { LoadPlugin(PluginPath+"ffms2.dll") } catch (_) { LoadCPlugin(PluginPath+"ffms2.dll") }
AudioDub(FFVideoSource("C:\Users\Plasma User\Documents\WebmforLazys\WebMConverter-3.39.0\testvid2.mp4",cachefile="C:\Users\Jonathan\AppData\Local\Temp\77-78-E9-4F-0A-18-72-CD-E0-91-47-6B-D2-C2-D5-79.ffindex",track=0), FFAudioSource("C:\Users\Plasma User\Documents\WebmforLazys\WebMConverter-3.39.0\testvid2.mp4",cachefile="C:\Users\Jonathan\AppData\Local\Temp\77-78-E9-4F-0A-18-72-CD-E0-91-47-6B-D2-C2-D5-79.ffindex",track=1))
# This is an AviSynth script. You may write advanced commands below, or just press the buttons above for smooth sailing.

**Comment by argorar** [comment]
Now that is a new error. Try this [WebMConverter.zip](https://github.com/argorar/WebMConverter/files/15262835/WebMConverter.zip), replace in your folder.
My theory is your new operative system. If fails again [enable long paths.](https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=powershell)

**Comment by PlasmaLover** [comment]
I have used the new WebMConverter.exe you sent me in the link and it has given me the same error. I have also enabled long paths and restarted my computer in case. I ran the WebMConverter.exe on admin and still have these errors. No luck.

![Screenshot 2024-05-09 122205](https://github.com/argorar/WebMConverter/assets/169137611/d272a534-9165-41f9-bdad-38d1451460dc)

![Screenshot 2024-05-09 121319](https://github.com/argorar/WebMConverter/assets/169137611/cd137a7f-42f0-46ba-a9cd-9e8a08602962)

![Screenshot 2024-05-09 120755](https://github.com/argorar/WebMConverter/assets/169137611/381ed2ff-1f2b-49f0-a783-84320ac45397)


====

Arguments for pass 1: -f nut -i pipe:0   -an -c:v libvpx-vp9 -pix_fmt yuv420p -threads 12 -slices 8 -metadata title="testvid" -lag-in-frames 16 -auto-alt-ref 1  -b:v 1389k -qcomp 0 -fs 4194304 -aq-mode 4 -row-mt 1 -tile-columns 6 -tile-rows 2 -pass 1 -passlogfile "C:\Users\Jonathan\AppData\Local\Temp\679d3571-c2b7-4d58-a41b-b636f3974ebe"  -f webm -y "NUL"
Arguments for pass 2: -f nut -i pipe:0   -an -c:v libvpx-vp9 -pix_fmt yuv420p -threads 12 -slices 8 -metadata title="testvid" -lag-in-frames 16 -auto-alt-ref 1  -b:v 1389k -qcomp 0 -fs 4194304 -aq-mode 4 -row-mt 1 -tile-columns 6 -tile-rows 2 -pass 2 -passlogfile "C:\Users\Jonathan\AppData\Local\Temp\679d3571-c2b7-4d58-a41b-b636f3974ebe"  -f webm -y "C:\Users\Plasma User\Documents\WebmforLazys\WebMConverter-3.39.0\testvid.webm"

--- CREATING AVISYNTH PROXY --- 
[avisynth @ 0284e4a0] Unable to load C Plugin: "C:\Users\Plasma User\Documents\WebmforLazys\WebMConverter-3.39.0\Binaries\Win32\ffms2.dll", error=0x7e
(C:\Users\Jonathan\AppData\Local\Temp\tmp4CC9.tmp, line 2)
(C:\Users\Jonathan\AppData\Local\Temp\tmp4CC9.tmp, line 2)
C:\Users\Jonathan\AppData\Local\Temp\tmp4CC9.tmp: Unknown error occurred
[nut @ 00000252fc89d280] No main startcode found.

[in#0 @ 00000252fc89d140] Error opening input: Invalid data found when processing input
--- FFMPEG HAS EXITED ---
--- ENTERING PASS 2 ---
--- CREATING AVISYNTH PROXY --- 
Error opening input file pipe:0.
Error opening input files: Invalid data found when processing input
[avisynth @ 00fce4a0] Unable to load C Plugin: "C:\Users\Plasma User\Documents\WebmforLazys\WebMConverter-3.39.0\Binaries\Win32\ffms2.dll", error=0x7e
[nut @ 0000026cfbd0d1c0] No main startcode found.
(C:\Users\Jonathan\AppData\Local\Temp\tmp4CC9.tmp, line 2)
--- FFMPEG HAS EXITED ---
[in#0 @ 0000026cfbd0d080] Error opening input: Invalid data found when processing input
(C:\Users\Jonathan\AppData\Local\Temp\tmp4CC9.tmp, line 2)
Error opening input file pipe:0.
C:\Users\Jonathan\AppData\Local\Temp\tmp4CC9.tmp: Unknown error occurred
Error opening input files: Invalid data found when processing input


ffmpeg.exe exited with exit code -1094995529. That's usually bad.
If you have no idea what went wrong, open an issue on Github and copy paste the output of this window there, also upload test video.

==========
Here is the Temp folder error:
==========

PluginPath = "C:\Users\Plasma User\Documents\WebmforLazys\WebMConverter-3.39.0\Binaries\Win32\"
try { LoadPlugin(PluginPath+"ffms2.dll") } catch (_) { LoadCPlugin(PluginPath+"ffms2.dll") }
AudioDub(FFVideoSource("C:\Users\Plasma User\Documents\WebmforLazys\WebMConverter-3.39.0\testvid.mp4",cachefile="C:\Users\Jonathan\AppData\Local\Temp\BF-E3-B9-4D-2D-17-7F-55-AB-8B-18-C4-3B-8C-54-F5.ffindex",track=0), FFAudioSource("C:\Users\Plasma User\Documents\WebmforLazys\WebMConverter-3.39.0\testvid.mp4",cachefile="C:\Users\Jonathan\AppData\Local\Temp\BF-E3-B9-4D-2D-17-7F-55-AB-8B-18-C4-3B-8C-54-F5.ffindex",track=1))
# This is an AviSynth script. You may write advanced commands below, or just press the buttons above for smooth sailing.

**Comment by PlasmaLover** [comment]
Do you think it would be best to wait for a windows update before checking again?

**Comment by argorar** [comment]
As last test with the new error  *Unable to load C Plugin: C:\Users\Plasma User\Documents\WebmforLazys\WebMConverter-3.39.0\Binaries\Win32\ffms2.dll* I would download zip file from releases and put in main disk. The error looks like caused externally.

**Comment by PlasmaLover** [comment]
I downloaded the Release Version 3.39.0 through here and selected WebMConverter-3.39.0.zip.
I put it into my C:\Disk which is my main here or have I done it wrong? 

![Screenshot 2024-05-12 222047](https://github.com/argorar/WebMConverter/assets/169137611/f3a1101a-0574-4195-b9b8-290840dd05d6)


=========

Arguments for pass 1: -f nut -i pipe:0   -an -c:v libvpx-vp9 -pix_fmt yuv420p -threads 12 -slices 8 -metadata title="testvid" -lag-in-frames 16 -auto-alt-ref 1  -b:v 2778k -qcomp 0 -fs 8388608 -aq-mode 4 -row-mt 1 -tile-columns 6 -tile-rows 2 -pass 1 -passlogfile "C:\Users\Jonathan\AppData\Local\Temp\e6b83013-8612-4855-9c8d-cbeb725a221f"  -f webm -y "NUL"
Arguments for pass 2: -f nut -i pipe:0   -an -c:v libvpx-vp9 -pix_fmt yuv420p -threads 12 -slices 8 -metadata title="testvid" -lag-in-frames 16 -auto-alt-ref 1  -b:v 2778k -qcomp 0 -fs 8388608 -aq-mode 4 -row-mt 1 -tile-columns 6 -tile-rows 2 -pass 2 -passlogfile "C:\Users\Jonathan\AppData\Local\Temp\e6b83013-8612-4855-9c8d-cbeb725a221f"  -f webm -y "C:\Users\Plasma User\Desktop\testvid.webm"

--- CREATING AVISYNTH PROXY --- 
[avisynth @ 027ee4a0] Unable to load C Plugin: "\\?\C:\PROGRA~1\WEBMCO~1.0\Binaries\Win32\ffms2.dll", error=0x7e
(C:\Users\Jonathan\AppData\Local\Temp\tmp5736.tmp, line 2)
(C:\Users\Jonathan\AppData\Local\Temp\tmp5736.tmp, line 2)
C:\Users\Jonathan\AppData\Local\Temp\tmp5736.tmp: Unknown error occurred
[nut @ 000001b4bfb1d380] No main startcode found.

--- FFMPEG HAS EXITED ---
--- ENTERING PASS 2 ---
--- CREATING AVISYNTH PROXY --- 
[in#0 @ 000001b4bfb1d240] Error opening input: Invalid data found when processing input
Error opening input file pipe:0.
Error opening input files: Invalid data found when processing input
[avisynth @ 0282e4a0] Unable to load C Plugin: "\\?\C:\PROGRA~1\WEBMCO~1.0\Binaries\Win32\ffms2.dll", error=0x7e
(C:\Users\Jonathan\AppData\Local\Temp\tmp5736.tmp, line 2)
[nut @ 000001efc7a3d440] No main startcode found.
--- FFMPEG HAS EXITED ---
(C:\Users\Jonathan\AppData\Local\Temp\tmp5736.tmp, line 2)
[in#0 @ 000001efc7a3d300] Error opening input: Invalid data found when processing input
C:\Users\Jonathan\AppData\Local\Temp\tmp5736.tmp: Unknown error occurred
Error opening input file pipe:0.

Error opening input files: Invalid data found when processing input

ffmpeg.exe exited with exit code -1094995529. That's usually bad.
If you have no idea what went wrong, open an issue on Github and copy paste the output of this window there, also upload test video.

===== 
Here the temp file too
=====

PluginPath = "\\?\C:\PROGRA~1\WEBMCO~1.0\Binaries\Win32\"
try { LoadPlugin(PluginPath+"ffms2.dll") } catch (_) { LoadCPlugin(PluginPath+"ffms2.dll") }
AudioDub(FFVideoSource("C:\Users\Plasma User\Desktop\testvid.mp4",cachefile="C:\Users\Jonathan\AppData\Local\Temp\BF-E3-B9-4D-2D-17-7F-55-AB-8B-18-C4-3B-8C-54-F5.ffindex",track=0), FFAudioSource("C:\Users\Plasma User\Desktop\testvid.mp4",cachefile="C:\Users\Jonathan\AppData\Local\Temp\BF-E3-B9-4D-2D-17-7F-55-AB-8B-18-C4-3B-8C-54-F5.ffindex",track=1))
# This is an AviSynth script. You may write advanced commands below, or just press the buttons above for smooth sailing.

**Comment by argorar** [comment]
I can't help you more since I can't replicate the issue.

**Comment by PlasmaLover** [comment]
I think I found a workaround. I believe there is a corruption in the file ffms2.dll in Version 3.39.0.
I have also tested ffms2.dll from  Version 3.34.0  and replaced it in the new one but did not work.

Here is what I did:
- WebMConverter  Version 3.39.0
- AviSynth (2.6.0, 32-bit)
- friend gave me his ffms2.dll from webm for ret*rds (I don't want github to ban me for spelling it out)
- replaced the ffms2.dll file in the directory linked

![Screenshot 2024-05-13 130535](https://github.com/argorar/WebMConverter/assets/169137611/640e8c7c-878d-41be-b70f-de8fe1ab3c7e)

![Screenshot 2024-05-13 132919](https://github.com/argorar/WebMConverter/assets/169137611/c5292c08-cd69-434e-969f-a5664391b725)
